tests/test_data/output_inline_grouped.txt

```diff
@@ -0,0 +1,31 @@
+"""Docstring here"""
+
+from __future__ import print_function, unicode_literals
+import datetime
+from os import path as ospath
+
+import something  # with comment
+from a import b
+from a.b import c, d
+from other import (  # noqa
+                   # lots of happy things below
+                   and_rainbows,
+                   something,  # something comment
+                   something_else)
+from other.package.subpackage.module.submodule import (  # noqa
```

Comment:
this statement comment should be combined used as first leaf comment, unless maybe first leaf has comments of its own.